### PR TITLE
Patch cuDNN v8.3.2/v8.4.0 build 0

### PR DESCRIPTION
**Pull request by leofang**
cc: @conda-forge/cudnn 

v8.3.2 build 1 fixed it (https://github.com/conda-forge/cudnn-feedstock/pull/46), and v8.4.0 build 1 is being done here (https://github.com/conda-forge/cudnn-feedstock/pull/45).
```
$ python show_diff.py                  
Downloading: https://conda.anaconda.org/conda-forge/noarch/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/noarch/repodata.json.bz2
noarch::dropbox-11.32.0-pyhd8ed1ab_0.tar.bz2
-    "python =2.7|>=3.4",
+    "python ==2.7.*|>=3.4",
Downloading: https://conda.anaconda.org/conda-forge/linux-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-64/repodata.json.bz2
linux-64::cudnn-8.3.2.44-hed8a83a_0.tar.bz2
-    "cudatoolkit >=11.2,<12",
linux-64::cudnn-8.4.0.27-hed8a83a_0.tar.bz2
-    "cudatoolkit >=11.2,<12",
Downloading: https://conda.anaconda.org/conda-forge/linux-armv7l/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-armv7l/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-aarch64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-aarch64/repodata.json.bz2
linux-aarch64::cudnn-8.3.2.44-h66ab864_0.tar.bz2
-    "cudatoolkit >=11.2,<12",
Downloading: https://conda.anaconda.org/conda-forge/linux-ppc64le/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-ppc64le/repodata.json.bz2
linux-ppc64le::cudnn-8.3.2.44-h321de51_0.tar.bz2
-    "cudatoolkit >=11.2,<12",
Downloading: https://conda.anaconda.org/conda-forge/osx-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-64/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-arm64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-arm64/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-32/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-32/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-64/repodata.json.bz2
win-64::cudnn-8.3.2.44-hf5f08ae_0.tar.bz2
-    "cudatoolkit >=11.2,<12",
win-64::cudnn-8.4.0.27-hf5f08ae_0.tar.bz2
-    "cudatoolkit >=11.2,<12",
```

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
Since there's another build already available, I don't think this is necessary. (We usually loosen pins for builds that don't have a corresponding build number fix with the correct loosened pin)

**Comment by leofang** [comment]
That's even better. I was under the impression that a repodata patch must be accompanied by a fix PR.

**Comment by leofang** [comment]
Thanks, Isuru.

**Comment by ngam** [comment]
> Since there's another build already available, I don't think this is necessary. (We usually loosen pins for builds that don't have a corresponding build number fix with the correct loosened pin)

But because of the dodgy work in the cudnn feedstock, there's this: https://anaconda.org/conda-forge/cudnn/files

a cudnn version of [8.4.0.27](https://anaconda.org/conda-forge/cudnn/8.4.0.27/download/linux-64/cudnn-8.4.0.27-hed8a83a_0.tar.bz2) which has the same "problem" that was "accidentally" updated 🤷

**Comment by leofang** [comment]
Yes I've told @Ivanyashchuk about it offline and he'll rectify it in https://github.com/conda-forge/cudnn-feedstock/pull/45, see my PR description.

**Comment by ngam** [comment]
> rectify

Fine with me, I am cool if we actually drop <11.2 anyway. 

> offline

I just wish you and your people communicate more openly instead of "offline" since as you'd say, this is a community project. It's often hard to follow what's going on with these nvidia packages, maybe because of this secretive and private communication, it is pretty frustrating to help or work with you on moving the ecosystem forward.
